specutils/tests/test_analysis.py

```diff
@@ -4,8 +4,12 @@
 import numpy as np
 import pytest
 
+from ..tests.spectral_examples import simulated_spectra
```

Comment:
This is *only* a "scalar" spectrum, right?  I.e. one spectral axis and one flux?  We do not this function to work for the one spectral axis, many-flux case... But perhaps it's best to get this in first and then worry about that?  (Or maybe @nmearl can advise on whether it still requires decisions to be made in `Spectrum1D`?)

Reply:
Yes, this is for a scalar spectrum only. I understood the many-flux case is not implemented yet.  If hte many-flux case is fully implemented then I should change this, if not, then I'll get this in first and more changes could be done in another PR.

Reply:
I think it's fine to leave addressing the many-fluxes one-dispersion case for a later PR.  Might be worth putting in the docstrings a note that this only works for the single-flux case, though?

Reply:
@nmearl Could you point me to the tests for many fluxes, one dispersion in specutils?  I would like to see how it is defined and used.